CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+## Contributing to Tasties project
+
+---
+
+Hello and welcome to Tasties project. 
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+
+
+
+### Quicklinks
+
+---
+
+- [Pre-Requirements](#pre-requirements)
+- [Issues](#issues)
+- [Pull Requests](#pull-requests)
+- [Testing](#testing)
+- [Code of Conduct](#code-of-conduct)
+- [Getting Help](#getting-help)
+
+
+### Pre Requirements
+
+---
+
+To contribute to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
```

Comment:
Maybe mention which version is most recommended to be used (hint, the one all of you are using)

Reply:
Also, even better to add direct links to the downloads so it will be easier for a new contributor to start